Gemfile

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Must have run rubocop with MRI? I have been wondering about leaving this fix in or not. Guess it doesn't hurt jruby.

Reply:
oh yeah at some point I ran rubocop with MRI - I did not realize these - I don't know if it'd hurt to have them with jruby 🤔 but it did not give me any errors when I index the sample with jruby 🤔

Reply:
👌